<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Sayı Tahmin Oyunu</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Aralık</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Aralık</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnedit_birinciAralik">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnedit_ikinciAralik">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buton_Uret">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Edilecek Sayıyı Üret</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buton_TahminEt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tahmin Et</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_sonuc">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>256</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
